presentation/common/src/test/_helpers/Content.ts

```diff
@@ -1,148 +1,169 @@
-  return new Descriptor({
-    connectionId: "",
-    displayType: "",
-    contentFlags: 0,
-    selectClasses: [createTestSelectClassInfo()],
-    categories: [createTestCategoryDescription()],
-    ...props,
-  });
-}
-
-export function createTestContentItem(props: {
-  primaryKeys?: InstanceKey[];
-  label?: LabelDefinition | string;
-  imageId?: string;
-  classInfo?: ClassInfo;
-  values: ValuesMap;
-  displayValues: DisplayValuesMap;
-  mergedFieldNames?: string[];
-  extendedData?: { [key: string]: any };
-}) {
-  return new Item(
-    props.primaryKeys ?? [createTestECInstanceKey()],
-    props.label ?? "",
-    props.imageId ?? "",
-    props.classInfo,
-    props.values,
-    props.displayValues,
-    props.mergedFieldNames ?? [],
-    props.extendedData,
-  );
-}
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import {
+  CategoryDescription, ClassInfo, Descriptor, DescriptorSource, DisplayValuesMap, EditorDescription, Field, InstanceKey, Item, LabelDefinition,
+  NestedContentField, PropertiesField, Property, PropertyValueFormat, RelationshipPath, RendererDescription, SelectClassInfo, StructTypeDescription,
+  TypeDescription, ValuesMap,
+} from "../../presentation-common";
+import { RelationshipMeaning } from "../../presentation-common/rules/content/modifiers/RelatedPropertiesSpecification";
+import { createTestECClassInfo, createTestECInstanceKey, createTestRelationshipPath } from "./EC";
+
+/**
+ * @internal Used for testing only.
+ */
+export const createTestCategoryDescription = (props?: Partial<CategoryDescription>) => ({
+  name: "test-category",
+  label: "Test Category",
+  description: "Test category description",
+  priority: 0,
+  expand: false,
+  ...props,
+});
+
+/**
+ * @internal Used for testing only.
+ */
+export const createTestSelectClassInfo = (props?: Partial<SelectClassInfo>) => ({
+  selectClassInfo: createTestECClassInfo(),
+  isSelectPolymorphic: false,
+  ...props,
+});
+
+/**
+ * @internal Used for testing only.
+ */
+export function createTestSimpleContentField(props?: {
+  category?: CategoryDescription;
+  type?: TypeDescription;
+  name?: string;
+  label?: string;
+  isReadonly?: boolean;
+  priority?: number;
+  editor?: EditorDescription;
+  renderer?: RendererDescription;
+}) {
+  return new Field(
+    props?.category ?? createTestCategoryDescription(),
+    props?.name ?? "SimpleField",
+    props?.label ?? "Simple Field",
+    props?.type ?? { valueFormat: PropertyValueFormat.Primitive, typeName: "string" },
+    props?.isReadonly ?? false,
+    props?.priority ?? 0,
+    props?.editor,
+    props?.renderer
+  );
+}
+
+/**
+ * @internal Used for testing only.
+ */
```

Comment:
Line endings

Reply:
The changes in these presentation test helper files are actually from adding these internal jsdocs 
![image](https://user-images.githubusercontent.com/11051042/136087263-db46e89c-d3ef-468a-bb16-e7a6ec9c33d3.png)

Reply:
Ah okay, my bad. Files must have blurred together while reviewing :)